### PR TITLE
add cap-add && cap-drop flags

### PR DESCRIPTION
**Pull request by fahedouch**
Signed-off-by: fahedouch <fahed.dorgaa@gmail.com>

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -200,6 +201,15 @@ var runCommand = &cli.Command{
 			Name:  "label-file",
 			Usage: "Read in a line delimited file of labels",
 		},
+		// linux capabilities
+		&cli.StringSliceFlag{
+        	Name:  "cap-add",
+        	Usage: "Add Linux capabilities",
+        },
+        &cli.StringSliceFlag{
```
Comment (AkihiroSuda) [inline run.go:209]: Please gofmt

File: run.go [hunk 2]
```diff
@@ -307,6 +319,35 @@ func runAction(clicontext *cli.Context) error {
...
+        var caps []string
+        for _, c := range capAdd {
+            if strings.ToUpper(c) == "ALL" {
+                continue
+            }
+            caps = append(caps, "CAP_"+strings.ToUpper(c))
+        }
+    	opts = append(opts, oci.WithCapabilities(caps))
+
+    	caps = []string{}
```
Comment (AkihiroSuda) [inline run.go:342]: Please avoid reusing caps variable here. This should be a new variable like capsDrop

**Comment by AkihiroSuda** [comment]
> Signed-off-by: fahedouch

Could you use full name?

File: run.go [hunk 3]
```diff
@@ -54,6 +54,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
+	"github.com/containerd/cri/pkg/util"
```
Comment (AkihiroSuda) [inline run.go:57]: Can we avoid CRI dependency?

Comment (fahedouch) [reply run.go]: ok I move it inside `nerdctl`

File: pkg/util/strings.go [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+   limitations under the License.
+*/
+
+package util
+
+import "strings"
+
+// InStringSlice checks whether a string is inside a string slice.
+// Comparison is case insensitive.
+func InStringSlice(ss []string, str string) bool {
```
Comment (AkihiroSuda) [inline pkg/util/strings.go:20]: nit: this should be in existing strutil.go . Also please clarify the origin of the code in the comment.

File: pkg/util/strings.go [hunk 5]
```diff
@@ -0,0 +1,27 @@
+/*
+   Copyright The containerd Authors.
```
Comment (fuweid) [inline pkg/util/strings.go:2]: missing nerdctl authors comment I think

File: pkg/util/strings.go (hunk 4 above)
Comment (fahedouch) [reply pkg/util/strings.go:20]: @AkihiroSuda  what do you mean by `existing strutil.go` ? in nerdctl project or in other project ?

Comment (AkihiroSuda) [reply pkg/util/strings.go:20]: https://github.com/AkihiroSuda/nerdctl/blob/master/strutil.go

**Comment by AkihiroSuda** [comment]
Merging. I'll address nits in a follow-up

**Comment by AkihiroSuda** [comment]
Tests for this is being added in https://github.com/AkihiroSuda/nerdctl/pull/56/files

**Pull request merged**